lib/Controller/TempAvatarController.php

```diff
@@ -111,7 +111,7 @@ public function postAvatar(): DataResponse {
 	/**
 	 * Delete your avatar as a user
 	 *
-	 * @return DataResponse<Http::STATUS_OK|Http::STATUS_BAD_REQUEST, array<empty>, array{}>
+	 * @return DataResponse<Http::STATUS_OK, null, array{}>|DataResponse<Http::STATUS_BAD_REQUEST, array{error: 'avatar'}, array{}>
```

Comment:
I think for a response you should avoid literal values.
If a another value can be returned later the current spec is not compatible later.

Reply:
well if we make it string instead of literals, it can not be the documentation anymore, as we need a second list of known error codes, and if we put that into the status code line it means psalm can not help us anymore to ensure the list is up to date.

Reply:
Ok makes sense, I didn't know you wanted to use it this way 👍